specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/stable/2021-03-07/CommunicationIdentity.json

```diff
@@ -0,0 +1,286 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CommunicationIdentityClient",
+    "description": "Azure Communication Identity Service",
+    "version": "2021-03-07"
+  },
+  "paths": {
+    "/identities": {
+      "post": {
+        "tags": [
+          "Identity"
+        ],
+        "summary": "Create a new identity.",
+        "operationId": "CommunicationIdentity_CreateIdentity",
```

Comment:
The "Identity" suffix on the method is redundant, it should be `CommunicationIdentity_Create`.  Also, if this `POST` operation can be used to modify an existing identity the name should be `CommunicationIdentity_CreateOrUpdate`.

Reply:
This endpoint doesn't update any value. There is a description for the optional body to explain the purpose. Create is more appropriate than CreateOrUpdate.